Dockerfile

```diff
@@ -0,0 +1,45 @@
+#
+# Builder image
+#
+
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.13 AS build-env
```

Comment:
@ASzc what is the motivation to use the OpenShift registry? Maybe we could just stick with `FROM golang:1.14.4 AS build-env` ?

Reply:
The same image was used in the [keycloak operator](https://github.com/keycloak/keycloak-operator/blob/master/Dockerfile#L1) recently. There's a direct productized equivalent available. It would make things easier to productise later.

The general golang images probably aren't that different, so we could use them instead.

Reply:
@ASzc I understand. But Louketo and the Keycloak Operator have completely different purposes, so I wouldn't get too attached to that. When productization comes, we can revisit, but for now I'd suggest us to try another approach.

Reply:
Switched to golang's Debian-based image